torch/nn/functional.py

```diff
@@ -1509,7 +1509,7 @@ def softmax(input, dim=None, _stacklevel=3, dtype=None):
     return ret
 
 
-def gumbel_softmax(logits, tau=1, hard=False, eps=1e-10, dim=-1):
```

Comment:
Why do these numbers need to be changed? Is type parsing mistaking them to be ints?

Reply:
Yep.